package.json

```
@@ -37,7 +37,7 @@
 
 		"test:coverage": "vitest run --coverage",
 		"test:unit": "vitest run",
-		"test:api-single": "hurl --continue-on-error --jobs 4 --max-time 300 --test --variables-file .dev.vars --variables-file .api-test.vars",
+		"test:api-file": "hurl --continue-on-error --jobs 4 --max-time 300 --test --variables-file .dev.vars --variables-file .api-test.vars",
 		"test:api": "npm run test:api-single -- --glob \"**/*.hurl\"",
```

```suggestion
		"test:api": "npm run test:api-file -- --glob \"**/*.hurl\"",
```

ProTip: when renaming things, use search and replace as much as possible, do global searches and use regex searches.